install/install.sh

```diff
@@ -92,14 +92,36 @@ checkJqInstalled() {
     fi
 }
 
+function gh_curl() {
+    if [ -z "$GITHUB_TOKEN" ]
+    then
+        curl \
+            $@
+    else
+        curl -H "Authorization: token $GITHUB_TOKEN" \
+            $@
+    fi
+}
+
+function gh_wget() {
+    if [ -n "$GITHUB_TOKEN" ]
```

Comment:
above using -z, this is using -n. Probably just use same for consistency.

Reply:
ah, right

Reply:
fixed